tools/make_webshots.py

```diff
@@ -24,11 +24,45 @@ def get_dandisets():
     return sorted(x['identifier'] for x in dandisets['results'])
 
 
-def process_dandiset(driver, ds):
+def login(driver, username, password):
 
-    def wait_no_progressbar(cls):
-        WebDriverWait(driver, 30).until(
-            EC.invisibility_of_element_located((By.CLASS_NAME, cls)))
+    driver.get("https://github.com/login")
+    WebDriverWait(driver, 30).until(
+        EC.presence_of_element_located((By.NAME, "commit")))
+    try:
+        driver.find_element_by_id("login_field").clear()
+        driver.find_element_by_id("login_field").send_keys(username)
+        driver.find_element_by_id("password").clear()
+        driver.find_element_by_id("password").send_keys(password)
+        driver.find_element_by_name("commit").click()
+    except Exception:
+        driver.save_screenshot("github-failure.png")
```

Comment:
Make it always generate not only if exception... Can't suggest multiple lines edit on phone